testing/fake-dom.js

```diff
@@ -239,18 +246,10 @@ export class FakeWindow {
     this.requestAnimationFrame = raf;
   }
 
-  /**
-   * @param {string} type
-   * @param {function(!Event)} handler
-   * @param {(boolean|!Object)=} captureOrOpts
-   */
+  /** polyfill addEventListener. */
```

Comment:
Why replace the type declarations? Were they causing errors?

Reply:
yes lint errors